grafana-formula/metadata/form.yml

```diff
@@ -90,3 +90,15 @@ grafana:
       $name: Kubernetes namespaces dashboard
       $help: Add dashboard for monitoring Kubernetes namespaces
       $default: False
+
+    add_ha_dashboards:
```

Comment:
Maybe for consistency make it singular like the other states: `add_ha_dashboard`?

Reply:
yes , originally were 3 make sense